gov/governance/proposal.gno

```diff
@@ -21,6 +21,11 @@ var (
 	latestProposalByProposer = make(map[std.Address]uint64)  // proposer -> proposalId
```

Comment:
replace map to avl

Reply:
https://github.com/gnoswap-labs/gnoswap/pull/448/commits/4af3dee33556940a149dccd4daca5faf07eded46
https://github.com/gnoswap-labs/gnoswap/pull/448/commits/9deb288d117c71107481127bde8fbecd823002d0